src/kOS.Safe/Compilation/KS/Compiler.cs

```diff
@@ -173,14 +173,9 @@ private void PreProcess(ParseTree tree)
         {
             ParseNode rootNode = tree.Nodes[0];
             TraverseScopeBranch(rootNode);
-            PreProcessUserFunctions(rootNode);
+            IterateUserFunctions(rootNode, IdentifyUserFunctions);
             PreProcessStatements(rootNode);
-        }
-
-        private void PreProcessUserFunctions(ParseNode node)
-        {
-            IterateUserFunctions(node, IdentifyUserFunctions);
-            IterateUserFunctions(node, PreProcessUserFunctionStatement);
```

Comment:
This was removed because I had to split up these two steps and do one of them BEFORE PreProcessStatements and one of them AFTER PreProcessStatements.